src/index.html

```diff
@@ -1,17 +1,178 @@
+
+        <label class="form-field">
+          How old are You?
+          <input 
+            type="number" 
+            min="12"
+            max="100"
+            name="age" 
+            autocomplete="off"
+            value="12"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input 
+            type="date" 
+            name="date-birth"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          I accept the term of the agreement
+          <input 
+            type="checkbox" 
+            name="agreement" 
+            value="yes" 
+            autocomplete="off" 
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="style-block">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input 
+            type="email" 
+            name="email" 
+            placeholder="email@example.com" 
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          Password:
+          <input 
+            type="password" 
+            name="password" 
+            minlength="8"
+            maxlength="25"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="style-block">
+        <legend>An intersting fact about you!</legend>
+
+        <div class="form-field">
+          Do you love cats?
+          <label>
+            <input 
+              type="radio" 
+              name="animal" 
+              value="Y"
+            >Yes
+          </label>
+
+          <label>
+            <input 
+              type="radio" 
+              name="animal" 
+              value="N"
+            >No
+          </label>
+        </div>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input 
+            type="color" 
+            name="color" 
+            value="black"
+          >
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input type="time" name="sleep">
+        </label>
+
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select name="car" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Honda">Honda</option>
+            <option value="Audi">Audi</option>
+            <option value=" "></option>
+          </select>
+        </label>
+
+        <label class="form-field">
+          How do you rate our work?
+          <input 
+            type="range" 
+            name="rate" 
+            min="0" 
+            max="5000" 
+            value="0" 
+            step="10"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="style-block">
+        <legend>Additional info:</legend>
+
+        <label class="form-field">
+          Comments:
+          <textarea name="comments">
```

Comment:
Please fix cursor start position
![image](https://user-images.githubusercontent.com/54936044/221545532-a85a0753-d889-40eb-b907-f885568772e6.png)

Reply:
You have indentations in the html. **ANYTHING** that appears between the opening/closing `<textarea>` tags becomes PART of the text area

Reply:
Could you try to fix it? If you have problems - feel free to ask for help in **fe_chat**